CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+## Contributing to Tasties project
+
+---
+
+Hello and welcome to Tasties project. 
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+
+
+
+### Quicklinks
+
+---
+
+- [Pre-Requirements](#pre-requirements)
+- [Issues](#issues)
+- [Pull Requests](#pull-requests)
+- [Testing](#testing)
+- [Code of Conduct](#code-of-conduct)
+- [Getting Help](#getting-help)
+
+
+### Pre Requirements
+
+---
+
+To contribute to our project please download and install the following:
+
+- Vagrant.
+- VirtualBox(or other VM that works with "Vagrant").
+
+After the installations type 'vagrant up' and automatically the development environment will set up.
+
+
+### Issues
+
+---
+
+Issues should be used to report problems with the library, request a new feature, or to discuss potential changes before a PR is created.
+Please make sure that before the PR is created, the issue must be created and after the PR is created, the PR must be linked to the issue relevant to it.
```

Comment:
This sentences are repeating themselves. Please rephrase it to make it more simplified a readable

Reply:
Also, you're mentioning here the acronym `PR` where you only introduce it later on in the next section...